src/mixins/name-request-mixin.ts

```diff
@@ -105,7 +155,7 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
    * Returns the Name Request's approved name.
    * @param nr the name request response payload
    */
-  private _getApprovedName (nr: any): string {
+  _getApprovedName (nr: any): string {
```

Comment:
The underscore was my code style to indicate that it's a local/helper method. If it's no longer private then please remove the underscore.

Reply:
Thank you, i wasn't sure of the original thoughts on why it was named in that manner :) Will do!